tests/e2e/run_all_e2e_tests.sh

```
@@ -0,0 +1,77 @@
+# This script runs all available e2e tests for MCP Fuzzer
+# Used in CI pipelines for comprehensive end-to-end testing
+
+set -e  # Exit on any error
+
+echo "🚀 Starting MCP Fuzzer E2E Test Suite"
+echo "========================================"
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
```

Shellcheck (0.11.0)

[warning] YELLOW appears unused. Verify use (or export if used externally).

(SC2034)